iml-agent/src/daemon_plugins/postoffice.rs

```diff
@@ -44,7 +44,7 @@ pub fn create() -> impl DaemonPlugin {
 
 /// Return socket address for a given mailbox
 pub fn socket_name(mailbox: &str) -> String {
-    let sock_dir = env::get_var("SOCK_DIR");
+    let sock_dir = env::get_var_else("SOCK_DIR", "/run/iml");
```

Comment:
Hardcoding a default doesn't seem like the right fix. We should separate the code under test from the configuration.

Reply:
I see two separate issues here:

1. The service shouldn't crash without an environment variable defined (i. e. there must be a default value, if possible).
2. Your point :) The tests should depend on the environment at all.

Let me take another look.